packages/wallet/src/redux/adjudicator-state/reducer.ts

```diff
@@ -59,6 +61,14 @@ const fundingReceivedEventReducer = (
   const { channelId } = action;
   return setBalance(state, channelId, action.totalForDestination);
 };
+const channelUpdateReducer = (state: AdjudicatorState, action: actions.ChannelUpdate) => {
```

Comment:
Is this related to storing the redux-store in storage?

Reply:
Yes, it's used to update the `adjudicator-state` on load for stored channels. We currently don't store the `adjudicator-state` so we need to initialize it on load from storage.